futures-util/src/try_stream/into_stream.rs

```diff
@@ -0,0 +1,48 @@
+use core::mem::PinMut;
+use futures_core::stream::{Stream, TryStream};
+use futures_core::task::{self, Poll};
+
+/// Stream for the [`into_stream`](super::TryStreamExt::into_stream) combinator.
+#[derive(Debug)]
+#[must_use = "streams do nothing unless polled"]
+pub struct IntoStream<St> {
+    stream: St,
+}
```

Comment:
`impl<St: Unpin> Unpin for IntoStream<St> {}`

Reply:
I just checked. The Unpin impl is missing in `IntoFuture`  as well.

@cramertj Do we need an `Unpin` impl here?

Reply:
Yeah I didn't add it since IntoFuture doesn't have one. Not sure what the practical implications are of either way.

Reply:
These Unpin impls aren't necessary since Unpin is an auto trait. You only need one in places where you want a more generous impl than the automatic one.